packages/website/pages/docs/reference/node-troubleshooting.mdx

```diff
@@ -6,13 +6,15 @@ You can ignore any WARN logs.
 
 ### Node error logs
 
-#### `error: "failed to insert new head to L2 execution engine: missing trie node"`
-
-Make sure that your RPC is pointing to an Sepolia archive node, and not a full node.
-
 #### `error: "daily request count exceeded, request rate limited"`
 
 Your RPC provider has a limit on the number of requests. You can try using a different provider that offers higher limits.
+INFURA https://www.infura.io/
+Alchemy https://auth.alchemy.com/
```

Comment:
usually people fail because of these RPC provider. could we say something like: "Use an RPC provider with higher limits or [run a Sepolia node](/docs/guides/run-a-node/run-a-sepolia-node) which has no limit.